services/docker-compose.local.yml

```diff
@@ -84,6 +80,10 @@ services:
         "log_line_prefix=%m [%p] %q%u@%d/%a ",
         "-c",
         'listen_addresses="*"',
+        "-c",
+        "max_connections=413",
+        "-c",
+        "shared_buffers=256MB"
```

Comment:
how did you determined  the values for max_connections and shared_buffers?

Reply:
413 is the same as AWS.
the other one I don't really know... using numbers I found here and there.